include/ck/utility/reduction_operator.hpp

```diff
@@ -150,6 +150,27 @@ struct AMax
     }
 };
 
+template <class T>
```

Comment:
after refactoring gridwise_gemm_reduce to accept a tuple of pointers for reduced tensors, ```PassThrough``` would no longer be needed

Reply:
ping

Reply:
4446797212c5d114d2e3a9612f43791aa12bb81e